ceremonies/src/tests.rs

```diff
@@ -1092,7 +996,7 @@ fn early_rewards_with_new_attest_attendees_extrinsic_with_one_missing_attestatio
 }
 
 #[test]
-fn early_rewards_does_not_work_with_one_missing_attestation() {
```

Comment:
the previous testing logic didn't cover the test scenario I expected from the name. I think it makes more sense to test a missing submission of attestations. alternatively, we could keep both tests, but I don't think this is necessary

Reply:
i think we should keep both tests, because those are two completely different things.
the old test could be renamed to `early_rewards_does_not_work_with_one_missing_attestation_per_submitter`

Reply:
Agreed